2-mandatory/4-people-i-know.js

```diff
@@ -402,16 +416,27 @@ This time, I only want the full names of the people who can multitask
 */
 
 let colleaguesWhoCanMultitask = [];
-
+let colleagues = friends.map((friend) => friend.colleagues);
+colleagues.map((colleague) =>
+  colleague.map((colleague) => {
+    if (colleague.skills.includes("Multi-tasking")) {
+      colleaguesWhoCanMultitask.push(colleague.name);
+    }
+  })
+);
```

Comment:
```suggestion
let colleagues = friends.map((friend) => friend.colleagues).flat();
colleagues.map((colleague) =>
    if (colleague.skills.includes("Multi-tasking")) {
      colleaguesWhoCanMultitask.push(colleague.name);
    }
);
```

Reply:
You're right , there was a redundancy in the code 
`friends.map((friend) =>
  friend.colleagues.map((colleague) => {
    if (colleague.skills.includes("Multi-tasking")) {
      colleaguesWhoCanMultitask.push(colleague.name);
    }
  })
);`